File: tablas javascript/mysqltablas.js
// Importar el módulo mysql2
const mysql = require('mysql2');

// Configurar la conexión a la base de datos
const connection = mysql.createConnection({
  host: 'localhost',         // Dirección del servidor MySQL
  user: 'root',        // Usuario de MySQL
  password: '', // Contraseña del usuario
});

// Conectar a la base de datos
connection.connect((err) => {
  if (err) {
    console.error('❌ Error al conectar a MySQL:', err);
    return;
  }
  console.log('✅ Conexión exitosa a MySQL');
});

File: tablas go/mysqltablas.go
package main

import (
	"database/sql"
	"fmt"
	"log"

	_ "github.com/go-sql-driver/mysql" // Importar el driver MySQL
)

func main() {
	// Configurar la conexión a MySQL
	dsn := "root:@tcp(localhost:3306)/"

	// Crear la conexión
	db, err := sql.Open("mysql", dsn)
	if err != nil {
		log.Fatal("❌ Error al conectar a MySQL:", err)
	}
	defer db.Close()

	// Verificar la conexión
	err = db.Ping()
	if err != nil {
		log.Fatal("❌ No se pudo conectar a MySQL:", err)
	}

	fmt.Println("✅ Conexión exitosa a MySQL")
}
